prow/jobs/test-infra/test-infra.jobs.yaml

```diff
@@ -0,0 +1,188 @@
+presubmits: # runs on PRs
```

Comment:
a lot of code here, can you use job templates as here: https://github.com/kyma-project/test-infra/blob/master/prow/jobs/kyma/components/ui-api-layer/ui-api-layer.jobs.yaml